.eslintrc.json

```diff
@@ -33,7 +33,8 @@
     "public/**",
     "smoke-tests/designer/features/*.feature",
     "*.md",
-    "*.eslintrc.json"
+    ".eslintrc.json",
+    "package.json"
```

Comment:
unimportant - are you still having issues with this? I've changed eslint to only look at .ts(x)/.js(x) apart from *.translation.json

Reply:
Lemme check

Reply:
you're right, it seems not to be a problem anymore, will remove it now